cachi2/core/package_managers/yarn_classic/workspaces.py

```diff
@@ -63,15 +63,15 @@ def _get_workspace_paths(workspaces_globs: list[str], source_dir: RootedPath) ->
     return [path.resolve() for glob in workspaces_globs for path in source_dir.path.glob(glob)]
 
 
-def _extract_workspaces_globs(
-    package: dict[str, Any],
-) -> list[str]:
-    """Extract globs from workspaces entry in package dict."""
-    # This could be an Array or an Array nested in an Object.
-    # Official docs mentioning the former:
-    #   https://classic.yarnpkg.com/lang/en/docs/workspaces/
-    # Official blog containing a hint about the latter:
-    #   https://classic.yarnpkg.com/lang/en/docs/workspaces/
```

Comment:
Please keep at least one reference! We are dealing with externally defined format so it is a good idea to refer to a spec. The second link was supposed to point to an actual blog post, but apparently something went wrong when I was copying and pasting it the last time. I'll try and dig up the actual link. _Edit_: here it is: https://classic.yarnpkg.com/blog/2018/02/15/nohoist/#how-to-use-it